cm_sdk/models/__init__.py

```diff
@@ -0,0 +1,110 @@
+import json
+from json import JSONEncoder
+from enum import Enum
+
+__author__ = 'e0d'
+
+
+class CloudManagerJSONEncoder(JSONEncoder):
```

Comment:
Storing the relevant data in some sort of container would also remove the need for this JSON serialization hack.